pkg/launcher/paths.go

```diff
@@ -39,7 +39,7 @@ func DefaultPath(path defaultPath) string {
 	if runtime.GOOS == "windows" {
 		switch path {
 		case RootDirectory:
-			return "C:\\Program Files\\Kolide\\Launcher-kolide-k2\\data"
+			return "C:\\ProgramData\\Kolide\\Launcher-kolide-k2\\data"
```

Comment:
this was a recent change I can re-run some tests with to verify- but with prior testing this has always worked out regardless of existing/new installation because only the new installations have the new path in launcher.flags

Reply:
ohh duh =) ... yea, I forgot this was assigned in launcher.flags